helm/scripts/sso-presync.sh

```
@@ -9,4 +9,8 @@ source "$(dirname "$0")/postsync-lib.sh"
 
 # Create configmap
 create_or_update_configmap sso-config \
-  --from-literal=admin_allowed_users="$admin_allowed_users" 
\ No newline at end of file
+  --from-literal=admin_allowed_users="$admin_allowed_users" \
+  --from-literal=keycloak_base_url="$keycloak_base_url" \
+  --from-literal=client_id="$client_id" \
+  --from-literal=client_secret="$client_secret" \
```

Shellcheck (0.11.0)

[warning] client_secret is referenced but not assigned.

(SC2154)